src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/PropertyPageResources.resx

```diff
@@ -246,4 +246,7 @@
   <data name="OKBtnHelpText" xml:space="preserve">
     <value>Create the profile and return to the debug property page with the new profile selected</value>
   </data>
+  <data name="chkNativeCodeDebuggingText" xml:space="preserve">
+    <value>Enable native code debugging</value>
```

Comment:
Accelerator key?

Though now that I think about it, can we reuse the same resource from non-Core? Then the translations will be the same? Or are those resources still in the other DLL in the VS tree?

Reply:
I asked Tom about this and he said not to worry about it - the loc folks should have a database to pull from

Reply:
As for the accelerator keys - nothing on the page has them so I will make a separate PR to add those